src/Indeed/Provider.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace SocialiteProviders\Indeed;
+
+use GuzzleHttp\RequestOptions;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    public const IDENTIFIER = 'INDEED';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopeSeparator = ' ';
+
+    protected $scopes = ['email', 'offline_access'];
```

Comment:
Do we need `offline_access` token by default? I assume we can omit this in the provider

Reply:
Hi @atymic, I don't know why but they require to have `offline_access` included when requesting for `email`

[https://docs.indeed.com/dev/docs/authentication-overview#scopes](https://docs.indeed.com/dev/docs/authentication-overview#scopes:~:text=Incremental%20authorization%20enables%20apps%20to%20request%20only%20the%20scopes%20that%20they%20need%2C%20and%20requires%20the%20offline_access%2C%20which%20maintains%20the%20permissions%20that%20your%20app%20has.%20For%20example%2C%20if%20your%20app%20requests%20the%20email%20scope%2C%20your%20app%20must%20also%20request%20the%20offline_access%20scope.)

Here is a screenshot from the above url.

<img width="656" alt="image" src="https://github.com/SocialiteProviders/Providers/assets/26641709/6eb9c1aa-c68c-4246-a6ae-bb40e4d0bf7d">

Reply:
Ahh ok.